### PR TITLE
Use same JUnit version used in quarkus BOM

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

**Review by geoand — COMMENTED** [review]
Good catch!

**Comment by gastaldi** [comment]
Thanks, feel free to approve the PR when possible :wink:

**Comment by geoand** [comment]
Oh, I didn't :man_facepalming:

**Review by geoand — APPROVED** [review]
Did now :)

**Review by gsmet — CHANGES_REQUESTED** [review]
We have `org.junit:junit-bom` in Dependabot. Should we add the artifact used here too?

**Comment by gastaldi** [comment]
@gsmet good idea, let me do that change, then Dependabot can update it as well

File: independent-projects/qute/pom.xml [hunk 1]
```diff
@@ -92,12 +100,6 @@
                 <artifactId>smallrye-reactive-streams-operators-1.0</artifactId>
                 <version>${version.smallrye-reactive-streams-operators}</version>
             </dependency>
-            <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter</artifactId>
-                <version>${version.junit}</version>
-                <scope>test</scope>
```
Comment (gsmet) [inline independent-projects/qute/pom.xml]: Can you check that every time this thing is used, it has the proper test scope?

Comment (gastaldi) [reply independent-projects/qute/pom.xml]: Good catch. Fixed the scope in the generator subproject

**Review by gsmet — APPROVED** [review]

**Pull request merged**